packages/client/codegen.yml

```diff
@@ -0,0 +1,9 @@
+schema: ../server/src/schema.graphql
+generates:
+  ./src/shared/__generated__/:
```

Comment:
for sure - just wondering, is this because its best/customary to have further code written deeper in src to be accessing stuff from parent, rather than a reach-n-grab-to-here-from-there free-for-all?

Reply:
This is generally the approach I've used on other projects. There's a case to be made for moving it under `shared`, but on non-trivial projects I usually also have an eslint rule to [not import from inside modules](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-internal-modules.md). This means `shared` needs to re-export everything under `shared/__generated__`, which can create naming problems.